src/main/java/gregtech/common/GT_Worldgen_GT_Ore_SmallPieces.java

```diff
@@ -42,7 +42,7 @@ public boolean executeWorldgen(World aWorld, Random aRandom, String aBiome, int
         if (!this.mRestrictBiome.equals("None") && !(this.mRestrictBiome.equals(aBiome))) {
             return false; //Not the correct biome for small ore
         }
-        if (!isGenerationAllowed(aWorld, aDimensionType, ((aDimensionType == -1) && (this.mNether)) || ((aDimensionType == 0) && (this.mOverworld)) || ((aDimensionType == 1) && (this.mEnd)) || ((aDimensionType == -28) && (this.mMoon)) || ((aDimensionType == -29) && (this.mMars)) || ((aDimensionType == -30) && (this.mAsteroid)) ? aDimensionType : aDimensionType ^ 0xFFFFFFFF)) {
+        if (!isGenerationAllowed(aWorld, aDimensionType, ((aDimensionType == -1) && (this.mNether)) || ((aDimensionType == 0) && (this.mOverworld)) || ((aDimensionType == 1) && (this.mEnd)) || ((aDimensionType == -28) && (this.mMoon)) || ((aDimensionType == -29) && (this.mMars)) ? aDimensionType : aDimensionType ^ 0xFFFFFFFF)) {

```

Comment:
Ah you're right, I didn't know it was configurable, thanks for pointing that out.